ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -126,6 +126,8 @@ Assign Vars According To Product
         Set Suite Variable  ${MODEL_REGISTRY_NAMESPACE}    rhoai-model-registries
         Set Suite Variable    ${IMAGE_PULL_PATH}        registry.redhat.io
         Set Suite Variable    ${KSERVE_IMAGE_PULL_PATH}        registry.redhat.io
+        Set Suite Variable    ${KUEUE_IMAGE_PULL_PATH}        registry.redhat.io
+        Set Suite Variable    ${DASHBOARD_IMAGE_PULL_PATH}        registry.redhat.io
```

Comment:
Dashboard follows `IMAGE_PULL_PATH` value right? So this is probably not needed.